lib/breathe_client.rb

```diff
@@ -56,5 +150,37 @@ def absences(after:)
         .data[:absences]
     end
     memo_wise :absences
+
+    def sicknesses(after:)
+      client
+        .sicknesses
+        .list(start_date: after, exclude_cancelled_sicknesses: true)
```

Comment:
It bugs me that this all on one line, but the equivalent for trainings is on two lines...